memo_wise.gemspec

```diff
@@ -1,10 +1,19 @@
 # frozen_string_literal: true
 
+# After loading the VERSION constant, save it to a variable to use in this
+# gemspec and then undefine it. This allows benchmarks to compare multiple
+# versions of MemoWise against each other without inadvertently sharing the same
+# VERSION constant.
+# NOTE: Other uses of this trick often switch `require_relative` to `load`, as
+# in https://github.com/simplecov-ruby/simplecov/issues/557#issuecomment-825171399,
+# but for our purposes `require_relative` seems to work fine. If any issues
+# arise, feel free to use `load` instead.
 require_relative "lib/memo_wise/version"
+gem_version = MemoWise.send(:remove_const, :VERSION)
```

Comment:
So the problem now is if using require_relative, and then removing the constant, the file will not be reloaded later, and the constant will remain removed.  I'm not sure how this could work @JacobEvelyn

Reply:
I guess I'm full circle, to it needs to use `load`, and can optionally use the `wrap` param if on a new enough version of Ruby.